salt/metalk8s/kubernetes/coredns/files/coredns-deployment.yaml.j2

```diff
@@ -4,7 +4,12 @@ kind: Deployment
 metadata:
   labels:
     k8s-app: kube-dns
-    kubernetes.io/name: "CoreDNS"
+    app: kube-dns
```

Comment:
```suggestion
    app: coredns
```

Reply:
Not sure you can change this, may break some dashboards.